src/order/conditionally_complete_lattice.lean

```diff
@@ -203,8 +203,9 @@ lemma cInf_upper_bounds_eq_cSup {s : set α} (h : bdd_above s) (hs : s.nonempty)
   Inf (upper_bounds s) = Sup s :=
 (is_glb_cInf h $ hs.mono $ λ x hx y hy, hy hx).unique (is_lub_cSup hs h).is_glb
 
-/--Introduction rule to prove that b is the supremum of s: it suffices to check that b
-is larger than all elements of s, and that this is not the case of any `w<b`.-/
+/--Introduction rule to prove that `b` is the supremum of `s`: it suffices to check that `b`
+is larger than all elements of `s`, and that this is not the case of any `w<b`.
+See `Sup_intro` for a version in complete lattices. -/
 theorem cSup_intro (_ : s.nonempty) (_ : ∀a∈s, a ≤ b) (H : ∀w, w < b → (∃a∈s, w < a)) : Sup s = b :=
```

Comment:
This name `cSup_intro` and the unnamed hypotheses are leftovers from a long time ago, when I was just starting Lean and using the Isabelle terminology and proof style. It would be a good idea to change them to more Lean-like names and styles. Something like: `cSup_eq_of_forall_le_of_forall_lt_exists_lt` is a little bit verbose, but much clearer I think.

Reply:
I've just changed the names. Do you think the proofs should also be changed to look more like Lean proofs ?

Reply:
The proof style is fine.